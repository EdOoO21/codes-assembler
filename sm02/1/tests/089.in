572 242 440 -470 -713 -295 745 -868 -694 734
