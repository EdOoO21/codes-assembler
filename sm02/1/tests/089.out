734
-694
-868
745
-295
-713
-470
440
242
572
